extract_single_asset.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python3
```

Comment:
Was this file intended to be comitted?

Reply:
I thought it might be useful for others doing object decomp for it to be in the repo so kept it in the commit but I would be happy to exclude it.

Reply:
ideally it would be an option passed to the current assets script. we've all just been too lazy to do it lol

Reply:
I've made it an option in extract_assets instead, hopefully that's fine.